### ISSUE TITLE
elevator-pitch + mdx

### ISSUE THREAD

**Issue by sw-yx** [greglobinski/gatsby-themes#58]
hi!

love elevator pitch :) :) :)

would love it even more if it had MDX!

have a nice day!

**Comment by greglobinski** [comment]
@sw-yx 
Thank you. 
I do not say no, but there is so little room for content on the screens that I can't imagine why you would like to use MDX. Could you please give me an example use case?

**Comment by sw-yx** [comment]
having an interactive component like a form :)

**Comment by greglobinski** [comment]
Yup, that's a good point, a small form could be placed on one of the screens. 

Added to the to-do list. Thank you @sw-yx
